worth2/main/views.py

```diff
@@ -3,14 +3,22 @@
 from django.core.urlresolvers import reverse
 from django.views.generic.base import TemplateView
 from django.views.generic.list import ListView
-from django.shortcuts import get_object_or_404, redirect
+from django.shortcuts import get_object_or_404, redirect, render
 
 from pagetree.generic.views import PageView
+import quizblock.models as qm
```

Comment:
Alright, yeah I don't really have an opinion on this. I'll change this to `import quizblock`, so it's obvious I'm using quizblock every time I use this module in this file.